style.css

```diff
@@ -83,3 +83,146 @@ ul{
     max-width: 200px;
     padding: 10px;
 }
+/* style works section */
+.works-section{
+    background-color: white;
+    color: black;
+    padding-left: 24px;
+}
+.header-text{
+    font-size: 40px;
+    font-family: 'Crete Round', serif;
+    font-weight: 600;
+    line-height: 52px;
+    line-height: 102%;
+    color: #1C1A19;
+    padding-top: 120px;
+    margin: 24px 0;
+}  
+.hl{
+    width: 327px;
+    /* top: 107px; */
+    height: 1px;
+    color: #212121;
+    margin: 24px 0;
+} 
+.main-work-section{
+    position: absolute;
+    z-index: 1;
+    margin-top: 70px;
+    margin-bottom: 24px;
+    width: 327px;
+    height: 237px;
+}
+/* work list */
+.work-header{
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    line-height: 108%;
+    align-items: left;
+    justify-content: center;
+    left: 24px;
+    padding: 24px 0;
+}
+.work-description{
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    line-height: 124%;
+    justify-self: left;
+    align-self: flex-start;
+    color: #091E42;
+    margin: 10px 0;
+}
+.tags-container{
+    display: flex;
+    flex-direction: row;
+    gap: 12px;
+    width: 297px;
+    
+}
+.tag{
+    background-color: white;
+    border: 2px solid;
+    color: #8993A4;
+    padding: 0.3rem 0.75rem;
+    height: 40px;
+    flex: 1 5 auto;
+    font-size: 15px;
+    line-height: 20px;
+    line-height: 110%;
+}
+.main-btn{
+    margin: 1rem 0 0.75rem 0;
+    background-color: #FF6B00;
+    color: white;
+    width: 124px; 
+    height: 48px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 3%;
+}
+/*  */
+.work-cards-container{
+    margin: 0 5px;
+    position: relative;
+    top: 650px;
+    display: grid;
+    grid-template-rows: 386px 386px 386px 386px 386px 386px;
+    justify-content: center;
+}
+.work-card{
+    background-image: url('images/work-section/Img Placeholder.png');
+    background-repeat: no-repeat;
+    background-size: 100%;
+    grid-column: 6;
+    display: flex;
+    flex-direction: column;
+}
+.card-header{
+    color: white;
+    margin-top: 2.5rem;
+    padding: 0 .75rem;
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+}
+.card-description{
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    padding: 0 .75rem;
+}
+.card-tags{
+    display: flex;
+    flex-direction: row;
+    margin-left: 0.75rem;
+    margin-top: 0.5rem;
+    gap: 12px;
+}
+.blur{
+    background-color: rgba(255, 255, 255, 0.24);
+    color: white;
+    border: none;
+}
+.btn-max-width{
+    width: 100%;
+    background-color: #FF6B00;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
```

Comment:
- [ ] Make sure to always follow the button styling in the Figma design template for all the cards in the work section. You must stick to the design as much as possible (e.g., font, colors, images, text, margins). Adjust the height of the buttons with `class="btn-max-width"` .

> Figma Design Template
![fig-card-dsn](https://user-images.githubusercontent.com/71140133/205411990-323656f9-1977-4ac9-8e56-ae0ece2fd9a3.PNG)

> Your Design
![card-btns](https://user-images.githubusercontent.com/71140133/205412004-43e3ef10-72b0-41a4-8da4-55c888318b08.PNG)

Reply:
Thank you for your feedback. I have changed the styling of the button specifically the height to match the Figma design.